cli/cmd/uninject.go

```diff
@@ -90,24 +96,26 @@ func (rt resourceTransformerUninject) transform(bytes []byte, options *injectOpt
 	return output, []injectReport{report}, nil
 }
 
+func (rt resourceTransformerUninjectSilent) transform(bytes []byte, options *injectOptions) ([]byte, []injectReport, error) {
```

Comment:
For consistency with the (very new) `inject` output, change this:
```go
report := injectReport{
	name: fmt.Sprintf("%s/%s", strings.ToLower(conf.meta.Kind), conf.om.Name),
}
```
to this:
```go
report := injectReport{
	kind: strings.ToLower(conf.meta.Kind),
	name: conf.om.Name,
}
```

validate:
```bash
$ curl -s https://run.linkerd.io/emojivoto.yml | bin/go-run cli inject - | bin/go-run cli uninject - > /dev/null

namespace "emojivoto" skipped
deployment "emoji" injected
service "emoji-svc" skipped
deployment "voting" injected
service "voting-svc" skipped
deployment "web" injected
service "web-svc" skipped
deployment "vote-bot" injected

 "namespace/emojivoto" skipped
 "deployment/emoji" uninjected
 "service/emoji-svc" skipped
 "deployment/voting" uninjected
 "service/voting-svc" skipped
 "deployment/web" uninjected
 "service/web-svc" skipped
 "deployment/vote-bot" uninjected
```

Reply:
will do :+1: